include/benchmark/benchmark.h

```diff
@@ -847,7 +840,7 @@ class BENCHMARK_EXPORT State {
   BENCHMARK_ALWAYS_INLINE
   void SetBytesProcessed(int64_t bytes) {
     counters["bytes_per_second"] =
-        Counter(static_cast<double>(bytes), Counter::kIsRate, Counter::kIs1024);
+        Counter(static_cast<double>(bytes), Counter::kIsRate, kIs1024);
```

Comment:
sorry, this was a mistake on my part.  kIs1024 should be here as part of the public API in benchmark.h .. i thought Counter was internal.  i'm so sorry.

Reply:
Ohh. Okay, I will redo the changes.

Reply:
So I just have to put the `OneK enum` back to its place?

Reply:
yes i think so, but keep the include from string util to benchmark.h